master/buildbot/newsfragments/gitpoller-ignore-missing.bugfix

```diff
@@ -0,0 +1 @@
+Git poller: add `--ignore-missing` argument to `git log` call to avoid `fatal: bad object` errors
```

Comment:
I'm not sure whether this is the correct formatting (MarkDown). Should it be reStructuredText, or plain text without any formatting?

Reply:
it is rst that is included eventually in the doc. The spelling and format is checked by CI...

Reply:
Ah. In that case, double backticks would have been better :)